docker-compose.override.yml

```diff
@@ -8,13 +8,13 @@ services:
     # Log printing via Rich is enhanced by a TTY
     tty: true
     env_file: ./dev/.env.docker-compose
-    deploy:
-      resources:
-        reservations:
-          devices:
-            - driver: nvidia
-              count: 1
-              capabilities: [gpu]
```

Comment:
Can you leave a note that this needs to be uncommented when we implement active learning (or whatever the technical term is)?

Reply:
Sure. I believe active learning is the technical term.